.github/workflows/npm-publish.yml

```diff
@@ -12,23 +12,18 @@ jobs:
     name: npm-publish
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
```

Comment:
is there a reason you removed the name?

Reply:
I removed the name to keep it consistent with `build.yml` and what the format is for the Java SDK actions. It is not necessary to remove it though.